spec/administrate/views/fields/belongs_to/_index_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "rails_helper"
+
+describe "fields/belongs_to/_index", type: :view do
+  let(:product) { create(:product) }
+  let(:product_path) { polymorphic_path([:admin, product]) }
+  let(:link) { "<a href=\"#{product_path}\">#{product.name}</a>" }
+  let(:associated_class) { "test_associated_class" }
+  let(:belongs_to) do
+    instance_double(
+      "Administrate::Field::BelongsTo",
+      associated_class: associated_class,
+      display_associated_resource: product.name,
+      data: product,
+    )
+  end
+
+  it "calls valid_action? with the associated class" do
```

Comment:
I think there is no need to check if specific functions are called in feature test